src/openrct2/park/ParkFile.cpp

```diff
@@ -507,6 +509,8 @@ namespace OpenRCT2
                 cs.ReadWrite(gWidePathTileLoopPosition);
 
                 ReadWriteRideRatingCalculationData(cs, gRideRatingUpdateState);
+
+                cs.ReadWrite(gIsAutosave);
```

Comment:
You will need to increase the target save version number and only read this value if the save file is >= the first version when you added the new field.